addon/options/options.js

```diff
@@ -0,0 +1,26 @@
+/* global browser */

```

Comment:
Environments came from `src/webextension/lib/environments.js`. They are now part of the options page as we no longer have access to `about:config` prefs.